presentation/src/main/java/jjh/movie/app/presentation/ui/home/adapter/BoxOfficeAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package jjh.movie.app.presentation.ui.home.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import jjh.movie.app.presentation.databinding.ItemBoxOfficeBinding
+import jjh.movie.app.presentation.ui.home.viewholder.BoxOfficeViewHolder
+
+class BoxOfficeAdapter(
+    private val detailClickListener: (String) -> Unit,
+    private val movieClickListener: (Int) -> Unit,
```

Comment:
요부분은 네이밍적으로 개선해볼 수 있을 것 같슴다~
```suggestion
    private val onDetailClick: (String) -> Unit,
    private val onMovieClick: (Int) -> Unit,
```
이런식으루

Reply:
혹시 왜 ClickListener라는 네이밍이 별론지 알 수 있을까용..?